tasks/main.yml

```diff
@@ -37,24 +37,30 @@
   tags:
       - xpack
 
-- include: elasticsearch-templates.yml
+- include: elasticsearch-template-copy.yml
   when: es_templates
   tags:
       - templates
 
 - meta: flush_handlers
 
+- include: elasticsearch-template-insert.yml
```

Comment:
I thought that loading templates was an idempotent task for elasticsearch, so no need to handle this in 2 steps.

By the way, if you want to load only if the copy happen, I think you have to use `load_templates|changed`